boards/arm/nrf5340dk_nrf5340/Kconfig

```diff
@@ -84,7 +35,7 @@ config BOARD_ENABLE_CPUNET
 	default y if (BT || NRF_802154_SER_HOST)
 
 config DOMAIN_CPUNET_BOARD
-	string
+	string "Board used by the CPUNET domain"
 	default "nrf5340dk_nrf5340_cpunet" if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS
```

Comment:
Same here. The whole option is defined under `if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS`.

Reply:
removed